pyaedt/modeler/cad/Primitives.py

```diff
@@ -275,45 +587,4908 @@ def version(self):
+        object_id : str, int, list
+            One or more object IDs whose name will be returned. A list can contain
+            both strings (object names) and integers (object IDs).
+        return_list : bool, option
+            Whether to return a list of the selections. The default is
+            ``False``, in which case a string of the selections is returned.
+            If ``True``, a list of the selections is returned.
+
+        Returns
+        -------
+        str or list
+           Name of the objects corresponding to the one or more object IDs passed as arguments.
+
+        """
+        if not isinstance(object_id, list):
+            object_id = [object_id]
+        objnames = []
+        for el in object_id:
+            if isinstance(el, int) and el in self.objects:
+                objnames.append(self.objects[el].name)
+            elif isinstance(el, int):
+                objnames.append(el)
+            elif isinstance(el, Object3d):
+                objnames.append(el.name)
+            elif isinstance(el, FacePrimitive) or isinstance(el, EdgePrimitive) or isinstance(el, VertexPrimitive):
+                objnames.append(el.id)
+            elif isinstance(el, str):
+                objnames.append(el)
+        if return_list:
+            return objnames
+        else:
+            return ",".join([str(i) for i in objnames])
+
+    @pyaedt_function_handler()
+    def split(self, objects, plane=None, sides="Both", tool=None, split_crossing_objs=False, delete_invalid_objs=True):
+        """Split a list of objects.
+        In case of 3D design possible splitting options are plane, Face Primitive, Edge Primitive or Polyline.
+        In case of 2D design possible splitting option is plane.
+
+        Parameters
+        ----------
+        objects : str, int, or list
+            One or more objects to split. A list can contain
+            both strings (object names) and integers (object IDs).
+        plane : str, optional
+            Coordinate plane of the cut or the Application.PLANE object.
+            The default value is ``None``.
+            Choices for the coordinate plane are ``"XY"``, ``"YZ"``, and ``"ZX"``.
+            If plane or tool parameter are not provided the method returns ``False``.
+        sides : str, optional
+            Which side to keep. The default is ``"Both"``, in which case
+            all objects are kept after the split. Options are ``"Both"``,
+            ``"NegativeOnly"``, and ``"PositiveOnly"``.
+        tool : str, int, :class:`pyaedt.modeler.cad.elements3d.FacePrimitive`or
+                :class:`pyaedt.modeler.cad.elements3d.EdgePrimitive`, optional
+            For 3D design types is the name, ID, face, edge or polyline used to split the objects.
+            For 2D design types is the name of the plane used to split the objects.
+            The default value is ``None``.
+            If plane or tool parameter are not provided the method returns ``False``.
```

Comment:
```suggestion
            For a 3D design, the name, ID, face, edge, or polyline used to split the objects.
            For a 2D design, the name of the plane is used to split the objects.
            The default value is ``None``.
            If a plane or tool parameter are not provided, the method returns ``False``.
```
What is the behavior when ``None`` is used?